.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,40 @@
-| Q             | A
-| ------------- | ---
-| Is bugfix?    | ✔️/❌
-| New feature?  | ✔️/❌
-| Breaks BC?    | ✔️/❌
+🚨 If you have encountered a SECURITY RELATED ISSUE with Yii, [use this form](https://www.yiiframework.com/security) to report any security issues you have encountered to us. ❗DO NOT use the issue tracker or discuss it in the public forum as this will cause more harm than help.
+
+❗ Remove sections that are not applicable
```

Comment:
```suggestion
<!--
🚨 If you have encountered a SECURITY RELATED ISSUE with Yii, [use this form](https://www.yiiframework.com/security) to report any security issues you have encountered to us. ❗DO NOT use the issue tracker or discuss it in the public forum as this will cause more harm than help.

❗ Remove sections that are not applicable
-->
```